tests/CMakeLists.txt

```diff
@@ -152,15 +156,17 @@ add_aktualizr_test(NAME uptane_serial SOURCES uptane_serial_test.cc ARGS ${PROJE
 
 add_aktualizr_test(NAME uptane_secondary SOURCES uptane_secondary_test.cc)
 
+add_aktualizr_test(NAME asn1 SOURCES asn1_test.cc ARGS ${PROJECT_BINARY_DIR} PROJECT_WORKING_DIRECTORY)
+
 add_executable(aktualizr_uptane_vector_tests uptane_vector_tests.cc)
 target_link_libraries(aktualizr_uptane_vector_tests aktualizr_static_lib ${TEST_LIBS} crypto)
 set(TEST_SOURCES ${TEST_SOURCES} uptane_vector_tests.cc)
 
-add_test(NAME test_uptane_vectors COMMAND ${PROJECT_SOURCE_DIR}/tests/run_vector_tests.sh
-         ${PROJECT_SOURCE_DIR}/tests/tuf-test-vectors)
-add_test(NAME test_valgrind_uptane_vectors COMMAND ${PROJECT_SOURCE_DIR}/tests/run_vector_tests.sh
-         ${PROJECT_SOURCE_DIR}/tests/tuf-test-vectors valgrind)
-add_dependencies(build_tests aktualizr_uptane_vector_tests)
+#add_test(NAME test_uptane_vectors COMMAND ${PROJECT_SOURCE_DIR}/tests/run_vector_tests.sh
+#         ${PROJECT_SOURCE_DIR}/tests/tuf-test-vectors)
+#add_test(NAME test_valgrind_uptane_vectors COMMAND ${PROJECT_SOURCE_DIR}/tests/run_vector_tests.sh
+#         ${PROJECT_SOURCE_DIR}/tests/tuf-test-vectors valgrind)
+#add_dependencies(build_tests aktualizr_uptane_vector_tests)
```

Comment:
Should the uptane_vector tests get added back in at some point, or were these intentionally left deactivated?